assets/css/Analog_clock.css

```
@@ -94,7 +101,48 @@ z-index: 10000;
   position: absolute;
   width: 2px;
   height: 150px;
-  background: red;
+  background: var(--sec-color);
   z-index: 12;
   border-radius: 6px 6px 0 0;
 }
+
+.mode{
+  position: absolute;
+  margin-top: 60vh;
+  align-content: center;
+}
+
+.checkbox {
+  opacity: 0;
+  position: absolute;
+}
+
+.label {
+  width: 50px;
+  height: 26px;
+  background-color: #fff;
+  display: flex;
+  border-radius: 50px;
+  align-items: center;
+  justify-content: space-between;
+  padding: 5px;
+  position: relative;
+  transform: scale(1.5);
+}
+
+.ball {
+  width: 20px;
+  height: 20px;
+  background-color: white;
+  box-shadow: 0 0 2px black;
+  position: absolute;
+  top: 2px;
```

```suggestion
  top: 3px;
```
for better alignment